examples/effects-and-handlers/Black-Jack.flix

```diff
@@ -0,0 +1,178 @@
+///
+/// Player Actions.
+///
+enum Action with ToString {
+    case Hit,
+    case Stand
+}
+
+///
+/// Player Action.
+///
+eff Play {
+    def hitOrStand(): Result[String,Action]
+}
+
+///
+/// Card Ranks.
+///
+enum Card with ToString, Eq {
```

Comment:
I see that suit is not _really_ used in Blackjack, but a typical video game would show suits nonetheless.